### PR TITLE
chore: Add @StackScribe to MAINTAINERS list

### PR DESCRIPTION
**Pull request by StackScribe**

### CODE REVIEW

File: helm-service/chart/values.yaml [hunk 1]
```diff
@@ -7,9 +7,9 @@ helmservice:
     enabled: true                            # Creates a Kubernetes Service for the helm-service
 
 distributor:
-  stageFilter: ""                            # Sets the stage this helm service belongs to
-  serviceFilter: ""                          # Sets the service this helm service belongs to
-  projectFilter: ""                          # Sets the project this helm service belongs to
+  stageFilter: ""                            # Stage to which this helm service belongs; default is all; comma-separated list to filter for set of stages
+  serviceFilter: ""                          # Service to which this helm service belongs; default is all; comma-separated list to filter for set of services
+  projectFilter: ""                          # Project to which this helm service belongs; default is all; comma-separated list to filter for set of projects
```
Comment (mowies) [inline helm-service/chart/values.yaml:12]: If you want to merge this change as well, please put it into a separate PR so that it will show up correctly in the changelog :)

**Comment by StackScribe** [comment]
@mowies I don't actually want to merge anything except adding my name to the CONTRIBUTORS.md and MAINTAINERS files.  But I can't seem to get a.clean branch where I can do just that.  When I go into my fork and try to fetch updates, it gives me a button to create a PR and merge -- is that normal for keptn//keptn or is my fork just screwed up?  I thought I added myself to CONTRIBUTORS a while ago but apparently not.  Aargh!  Any advice as to how I can clear this all up?

**Review by oleg-nenashev — COMMENTED** [review]
Is there a counterpart GitHub Issue in https://github.com/keptn/community/issues ?

**Comment by mowies** [comment]
This PR needs a rebase...

**Comment by oleg-nenashev** [comment]
@StackScribe I will close it since there is no activity. Feel free to reopen
